app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] It would be better if our Post model has a method that `updates the post_counter for a user`